### ISSUE TITLE
https status of tests computed by runner and not in manifest

### ISSUE THREAD

**Issue by gsnedders** [web-platform-tests/wpt#12379]
For the same reason as #11285, we should move this into the manifest to avoid having logic in every runner.

**Comment by jgraham** [comment]
Yes, this is a good idea (honestly, I thought it already worked like this and was surprised when it didn't) and isn't very hard to do; just add a `is_https` flag on the `SourceFile` object, add a `https` parameter to `ManifestItem` and arrange for that to be stored/serialized in the manifest in the extra properties dictionary (I suggest defaulting to `False`).

**Comment by foolip** [comment]
FWIW, I'm now having to fix Blink's runner when importing the recent changes, so this would be nice :)

**Comment by mdittmer** [comment]
_friendly ping_; this is roadmap, but doesn't have an owner :thinking:

**Comment by foolip** [comment]
I'll downgrade to backlog, it doesn't seems like a particularly serious problem, more a nice-to-fix.

**Comment by schalkneethling** [comment]
@foolip If this is still valuable I would love to volunteer to take this as my first contribution. Let me know what the next steps would be. Thank you in advance.

**Comment by schalkneethling** [comment]
Also seems like this one will go hand in hand with https://github.com/web-platform-tests/wpt/issues/11285 mentioned by @gsnedders in the original issue description here.
